packages/central-server/app/subCommands/provision.js

```diff
@@ -15,9 +16,42 @@ import { initDatabase } from '../database';
 import { checkIntegrationsConfig } from '../integrations';
 import { loadSettingFile } from '../utils/loadSettingFile';
 import { referenceDataImporter } from '../admin/referenceDataImporter';
+import { normaliseSheetName } from '../admin/importer/importerEndpoint';
 import { getRandomBase64String } from '../auth/utils';
 import { programImporter } from '../admin/programImporter/programImporter';
 
+/**
+ * Validates that a reference data file contains all sheets importable through the reference data importer
+ * @param {string|Buffer} fileOrData - File path or data buffer
+ * @param {boolean} isData - Whether the first parameter is data (true) or file path (false)
+ */
+function validateFullReferenceDataImport(fileOrData, isData = false) {
```

Comment:
```suggestion
function validateFullReferenceDataImport(fileOrData) {
```
I might have missed something but it looks like this argument is not used at the moment, right?